### DISCUSSION TITLE
What is master-server?

### DISCUSSION THREAD

**Discussion by bseditor** [efroemling/ballistica#426] [Q&A] [answered]
Heard this term much but never came to know about what this is about?

**Comment by efroemling** [comment, marked as answer]
It's the central server that stores data for all accounts, manages all the tournaments, maintains lists of joinable games, etc. The game talks to it quite a bit in the background. It's technically not a single server (the actual number of machines it is running on varies based on how busy things are). There's also 2 master-servers now; the one that's been running for the past 10 years and a second shiny new one that I'm slowly transitioning some things to (the 'v2' master server).

**Comment by MatiasMGz** [comment]
An excelent answer! So... the 'v2' master server, at some point, would be the only server, right?
